content/applications/hr/lunch/products.rst

```diff
@@ -0,0 +1,61 @@
+========
+Products
+========
+
+No products are pre-configured in Odoo's *Lunch* app. Before any orders can be placed, the
+individual products that are being offered must first be configured.
+
+First, navigate to the products page by going to :menuselection:`Lunch app --> Configuration:
+Products`. Next click the :guilabel:`New` button in the top-left corner and a blank product form
+loads.
+
+Enter the following information on the form:
+
+- :guilabel:`Product Name*`: enter the name for the product.
+- :guilabel:`Product Category*`: using the drop-down menu, select the :ref:`category
+  <lunch/product-categories>` this product falls under.
+- :guilabel:`Vendor*`: using the drop-down menu, select the vendor that supplies this product.
+- :guilabel:`Price`: enter the price for the product. The currency is determined by the localization
+  of the company.
+- :guilabel:`Description`: enter a description of the product in this field. This description
+  appears beneath the product photo when users are viewing the options for the day.
+- :guilabel:`New Until`: using the calendar popover, select the date that the product will no
+  longer be labeled as new. Until this date, a green `New` tag appears on the product.
+- :guilabel:`Company`: if the product should only be available to a specific company, select it from
+  the drop-down menu. If this field is left blank, this product is available for all companies in
+  the database.
+- Image: hover over the image box in the top-right corner of the form, and click the
+  :icon:`fa-pencil` :guilabel:`(pencil)` icon that appears. A file explorer pop-up window appears.
+  Navigate to the image, then click :guilabel:`Open`.
+
+  (*) indicates required field.
```

Comment:
I would move this *above* the list.
Side note- we don't have a standard way of dealing with what are required fields vs not, and we probably should. I think this is fine, but I want to leave a comment here so KC and Sam can weigh in on this too.

Reply:
I used to do 'required' and 'optional' sections which was not ideal, and I recall someone suggesting this format to identify the required fields for such a long list. If it's just 2-3 items out of a list of 10-20, then I would have a line saying those were required, but this was too much. I am open to whatever suggestion or decision we agree on since I just want to let people know what they must fill in and wat can be eft out.